packages/react/fast-components-react-base/src/data-grid/README.md

```diff
@@ -0,0 +1,59 @@
+# Data grid
+The *data grid* component enables a display of tabular data in a scrolling grid with fixed column headers. 
+
+## Usage
+
+Authors provide the data the component will display through the "gridData" prop.  This data must take the form of an array of objects, each object in the array corresponds to one row of data in the grid.  Additionally, each object in the array must have a property which uniquely identifies it that is passed to the component through the 'dataRowKey' prop. 
+
+A simple data set using a "clientId" attribute as a unique identifier  could look like this.
+
+myData = [
```

Comment:
This should probably be code-formatted. Same for the reset of the code examples that are not code formatted.